# HG changeset patch
# Parent f8ea2e12ed5ab33f160e65f22c9ce8f379863053
tools/makefile: Add build target

The root Makefile has a build target which calls "$(MAKE) build" in each of xen/
tools/ stubdom/ and docs/, which fails because of the tools/ Makefile.

This patch adds a phony build target, allowing a call to "make build" in the
repository root directory still work.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>

diff -r f8ea2e12ed5a tools/Makefile
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -64,6 +64,9 @@ endif
 .PHONY: all
 all: subdirs-all
 
+.PHONY: build
+build: subdirs-all
+
 .PHONY: install
 install: subdirs-install
 	$(INSTALL_DIR) $(DESTDIR)/var/xen/dump
